webdriverio_taf/po/components/cart.component.js

```diff
@@ -0,0 +1,43 @@
+import { step } from '../../support/helpers'
+
+class Cart {
+  constructor (browser) {
+    this.browser = browser
+  }
+
+  get rootEl () {
+    return this.browser.$('[class^="Cart__Container"]')
+  }
+
+  get emptyMessageBlock () {
+    return this.rootEl.$('div[class^="CartProducts__CartProductsEmpty"]')
+  }
+
+  get checkoutButton () {
+    return this.rootEl.$('button=Checkout')
```

Comment:
let's not hardcode labels and put them in constants instead
```suggestion
    return this.rootEl.$(`button=${CHECKOUT}`)
```